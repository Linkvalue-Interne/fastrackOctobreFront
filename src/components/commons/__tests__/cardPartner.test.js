import React from 'react';
import renderer from 'react-test-renderer';
import serializer from 'jest-emotion';

import CardPartner from '../cardPartner';

expect.addSnapshotSerializer(serializer);

describe('cardPartnerCSS', () => {
  it('renders with correct styles', () => {

    const mock = {
      colors: { partnerBg: '#111' },
    };

    const tree = renderer.create(<CardPartner theme={mock} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
