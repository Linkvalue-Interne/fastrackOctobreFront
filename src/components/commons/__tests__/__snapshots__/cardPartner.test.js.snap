// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cardPartnerCSS renders with correct styles 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  height: 35vh;
  min-height: 300px;
  margin-bottom: 8rem;
  background: #111;
  border-radius: 45px;
  position: relative;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding-top: 3rem;
}

<div
  className="emotion-0"
/>
`;
